src/screens/Notifications/NotificationScreen.tsx

```diff
@@ -36,10 +36,34 @@ export class NotificationScreen extends Component<Props> {
     !!email && wallets.length && checkSubscription(wallets, email);
   }
 
-  onChangeEmailPress = () =>
-    this.props.navigation.navigate(Route.ChangeEmail, {
-      email: this.props.email,
+  onChangeEmailPress = () => {
+    this.props.navigation.navigate(Route.AddNotificationEmail, {
+      title: i18n.notifications.notifications,
+      subTitle: i18n.notifications.changeEmailTitle,
+      description: i18n.notifications.changeEmailDescription,
+      isBackArrow: true,
+      additionalContent: (
```

Comment:
could you extract additionalContent and onSuccess to variables? Would become more readable